### PR TITLE
Fix ResizeObserver loop limit exceeded

### PR DESCRIPTION
**Pull request by stephanussusanto**
@bitriddler 
After the update on the react measure - https://github.com/bitriddler/react-items-carousel/pull/7 
Based on the fixes on the react measure : 
https://github.com/souporserious/react-measure/pull/118/commits/9354ea705b4762e261cd7ff724eaf3bdd92d735e 
It move the setState under animationFrame.

However, there is a setState also under OnResize callback.
so, I need to move the setState under requestAnimationFrame.

### CODE REVIEW

**Comment by stephanussusanto** [comment]
@bitriddler  Can we merge this PR? It will be great if we can get a npm update also.
Thanks...

**Comment by matticus1181** [comment]
@bitriddler My app is also having this issue, so it would be great if this PR could be accepted. Thank you!

File: src/ItemsCarousel/PlaceholderCarousel.js [hunk 1]
```diff
@@ -44,13 +44,15 @@ export class PlaceholderCarousel extends React.Component {
...
-
+    let self = this;
     return (
       <Measure
         bounds
         margin={false}
         onResize={({ bounds }) => {
-          this.setState({ containerWidth: bounds.width });
+          requestAnimationFrame(function () {
+            self.setState({ containerWidth: bounds.width });
```
Comment (kareemaly) [inline src/ItemsCarousel/PlaceholderCarousel.js]: Maybe use an arrow function and remove the self variable ?
```
requestAnimationFrame(() => this.setState({ containerWidth: bounds.width }));
```

**Review by kareemaly — COMMENTED** [review]
Thanks @stephanussusanto for your PR! I'll merge as soon as you solve this small comment in both files 🚀

File: src/ItemsCarousel/PlaceholderCarousel.js (hunk 1 above)
Comment (stephanussusanto) [reply src/ItemsCarousel/PlaceholderCarousel.js]: @bitriddler this is fixed

**Pull request merged**